helper/ipsec.c

```diff
@@ -0,0 +1,157 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(C) 2020 Marvell.
```

Comment:
This should include:
Copyright (c) 2017-2018, Linaro Limited

We do not want to remove copyright attribution of existing code just because we move it to a different file.

Reply:
Ack. Keeping "Copyright (c) 2017-2018, Linaro Limited" in the new file.